internal/cloud/realm/access_list.go

```diff
@@ -83,10 +83,9 @@ func (c *client) AllowedIPCreate(groupID, appID, address, comment string, useCur
 	return allowedIP, nil
 }
 
-// TODO(REALMC-9689): Update to PATCH
 func (c *client) AllowedIPUpdate(groupID, appID, allowedIPID, newAddress, newComment string) error {
 	res, err := c.doJSON(
-		http.MethodPut,
+		http.MethodPatch,
```

Comment:
is there no corresponding test case that would need to be updated for this change?

Reply:
they are implemented but are skipped for now!

Reply:
ah thats totally my bad, I thought we were in `baas` 🙄 ... carry on